tests/kernel/timer/timer_api/testcase.yaml

```diff
@@ -4,6 +4,8 @@ tests:
       - kernel
       - timer
       - userspace
+    platform_exclude:
+      - nrf54h20dk/nrf54h20/cpuppr
```

Comment:
That's frustrating.  Can you file a bug on this, we should probably look at splitting this into two tests then.  timer_api is one of the core/most-important/best-validation-smokes tests and we really don't want to be disabling it for technicalities.

Reply:
Also, and I swear I'm not pointing fingers here, failures in timer_api are often very subtle things that look like "the test is wrong" or "it's just random/unfixable".  It's happened again and again that tiny edge case failures on obscure platforms or configurations turn out to be evidence of bugs in the driver or timer subsystem.  We really want this test to be on for everyone.